src/test/java/resources/testng/runAddAndSubtractTests.xml

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```

Comment:
I think you misplace `resources` folder
It should have path `src/test/resources/`, but you have `src/test/java/resources/`

Reply:
Done